cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/MonitorUtils.java

```diff
@@ -479,6 +498,9 @@ static void populatePartitionLoad(Cluster cluster,
                                     valuesAndExtrapolations.windows());
         needToAdjustCpuUsage = false;
       }
+      if (!deadBrokers.isEmpty()) {
+        LOG.info("Assign empty capacity to dead brokers {} in cluster model.", deadBrokers);
```

Comment:
Can we update the message to make it clear that these are not necessarily all the dead brokers, but only the dead brokers, whose capacity cannot be obtained from the capacity resolver?